website/docs/components/toast/partials/code/how-to-use.md

```diff
@@ -41,7 +40,7 @@ A different color can be applied to the toast using the `color` argument. This w
 
 ### Icon
 
-A different icon can be used in the toast using the `icon` argument.
+A different icon can be used in the alert using the `icon` argument. This accepts any [Helios icon](/foundations/icon/library) name.
```

Comment:
Curious why this is alert rather than Toast?

Reply:
Cause I copy & pasted it from the Alert documentation and forgot to update it 🙃  Will fix.